app/Http/Controllers/Api/UsersController.php

```diff
@@ -530,40 +531,16 @@ public function update(SaveUserRequest $request, $id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function destroy($id)
+    public function destroy(DeleteUserRequest $request, $id)
     {
         $this->authorize('delete', User::class);
-        $user = User::with('assets', 'assets.model', 'consumables', 'accessories', 'licenses', 'userloc')->withTrashed();
+        $user = User::with('assets', 'assets.model', 'consumables', 'accessories', 'licenses', 'userloc');
         $user = Company::scopeCompanyables($user)->find($id);
         $this->authorize('delete', $user);
```

Comment:
If you wanted to, you could move this logic to the DeleteUserRequest.

Reply:
> I did mention that, if you wanted, you could consider moving the ->authorize() calls up into the FormRequest if you wanted to - but that's not a dealbreaker for me at all. Thanks for doing this!

We don't do that anywhere else, and it seemed too big a leap for this PR. We'd need to extensively test anything that touches permissions.